doc_src/pages/examples/styling.njk

```diff
@@ -213,4 +213,4 @@ new TomSelect('#input-group-first',{create:true});
 {% endset %}
 </script>
 
-{{ demo_col('input-group', left_col, html, script) }}
+{{ demo_col('input-group-first', left_col, html, script) }}
```

Comment:
The new `IdAttributePlugin` discovered that this produced a duplicate ID. I updated the name to match that in the script above.